scripts/machine_translation/train_gnmt.py

```diff
@@ -170,10 +170,10 @@ def evaluate(data_loader):
         tgt_valid_length = tgt_valid_length.as_in_context(ctx)
         # Calculating Loss
         out, _ = model(src_seq, tgt_seq[:, :-1], src_valid_length, tgt_valid_length - 1)
-        loss = loss_function(out, tgt_seq[:, 1:], tgt_valid_length - 1).mean().asscalar()
+        loss = loss_function(out, tgt_seq[:, 1:], tgt_valid_length - 1).sum().asscalar()
```

Comment:
@liuzh91 I think `loss_function(...).mean() * (tgt_seq.shape[1] - 1)` is equivalent to `loss_function(...).sum()`